pulse2percept/retina.py

```diff
@@ -472,36 +469,72 @@ def calc_per_electrode(self, pt):
 
         return pt_entry
 
-    def calc_per_pixel(self, ecs_item, pt, layers, dojit=False):
-        # Convert the current map to one that includes axon streaks
-        ecm = effectivecurrentmap(ecs_item, pt, self.tsample)
-
-        # Run the model cascade
-        sr = self.model_cascade(ecm, layers, dojit=dojit)
+    def calc_per_pixel(self, ecs_item, pt_list, layers, dojit=False):
+        """Calculations to be done on a per-pixel basis
 
-        return sr
+        The following calculations need to be done on a pixel-by-pixel basis.

```

Comment:
This repeats what was said right above. I propose to change the first line of the docstring to say what these calculations represent.